Runtime/Scripts/MeshSyncInstanceRenderer.cs

```diff
@@ -15,7 +15,9 @@ internal class MeshSyncInstanceRenderer : MonoBehaviour
 
         [HideInInspector]
         [SerializeField] private Matrix4x4[] transforms;
-        [SerializeField] private GameObject reference; 
+        [SerializeField] private GameObject reference;
+        [SerializeField] private Transform  instanceParent;
```

Comment:
It seems that we are just setting this without using it.
I guess it will be used in later PRs ?

Reply:
I've removed this, turns out we don't need it.